sentry-android-fragment/src/main/java/io/sentry/android/fragment/SentryFragmentLifecycleCallbacks.kt

```diff
@@ -91,4 +117,45 @@ class SentryFragmentLifecycleCallbacks(
     private fun getFragmentName(fragment: Fragment): String {
         return fragment.javaClass.simpleName
     }
+
+    private fun isRunningSpan(fragment: Fragment): Boolean =
+        fragmentsWithOngoingTransactions.containsKey(fragment)
+
+    private fun startTracing(fragment: Fragment) {
+        if (!isPerformanceEnabled || isRunningSpan(fragment)) {
+            return
+        }
+
+        var transaction: ISpan? = null
+        hub.configureScope {
+            transaction = it.transaction
+        }
+
+        val fragmentName = getFragmentName(fragment)
+        val span = transaction?.startChild(FRAGMENT_LOAD_OP, fragmentName)
+
+        span?.let {
+            fragmentsWithOngoingTransactions[fragment] = it
+        }
+    }
+
+    private fun stopTracing(fragment: Fragment) {
```

Comment:
Any chance this isn't called ever? I wonder if it's possible to leak fragments in `fragmentsWithOngoingTransactions`

Reply:
we follow the fragment's lifecycle, same with the Activity lifecycle, if the methods don't get called, it's a bug in the androidx library.